### ISSUE TITLE
Images stops rendering at a certain point.

### ISSUE THREAD

**Issue by xZyph** [eKoopmans/html2pdf.js#450]
I have made a generator that create a PDF based on chosen products. When selecting 10-20 products the generator works just fine and there is no issue at all. But when I increase the amount of products, at a certain point the generator just stops rendering pictures halfway through. The HTML and everything else is rendering just fine, but the images below a certain point are just not visible at all. The images usually stops rendering at page 10.

I've googled this issue for ages and have found nothing specific related to my issue. The document does not exceed the theoretical max pixel sixe, and the logger shows all images are loaded into the application. I also do not recieve any error at all (which I've seen people get when filling up the heap memory).

I figured it could still be that the images are too big, but when fetching them through a proxy that reduces size, they still stop loading at the same point, even though the size is reduced by a massive amount.

Any tips on where to continue debugging this would be greatly appreciated.

**Comment by woodwoerk** [comment]
I'm experiencing the same issue and it seems to be in `html2canvas`. It's not really a solution, but decreasing the `scale` value allows more pages to be rendered without missing images (at the cost of image quality).

e.g. in your options config: 
```
  ...
  html2canvas:  { scale: 1 },
  ...
```

This [unsolved SO question](https://stackoverflow.com/questions/65917940/missing-images-depending-on-scale-in-html2canvas-html2pdf) details the same bug.

**Comment by xZyph** [comment]
> I'm experiencing the same issue and it seems to be in `html2canvas`. It's not really a solution, but decreasing the `scale` value allows more pages to be rendered without missing images (at the cost of image quality).
> 
> e.g. in your options config:
> 
> ```
>   ...
>   html2canvas:  { scale: 1 },
>   ...
> ```

Yeah, I've tried messing around with the scale, but it kills the quality and doesn't seem to get the results that I want.
I really hope I can fix this issue rather than rewriting the whole application to work from the backend instead. One thing I've learnt though, never ever try to do this in frontend xD
